public/js/controllers/homepage_controller.js

```diff
@@ -72,7 +72,7 @@ export default class extends Controller {
     })
     ws.registerEvtHandler('getmempooltxsResp', (evt) => {
       var m = JSON.parse(evt)
-      this.mempool.mergeMempool(m)
+      this.mempool.replace(m)
```

Comment:
Looking over it, I think I should have done this from the beginning. If we are getting the full copy of `MempoolInfo`, there is not reason to try to merge anything. Just replace it all.

Reply:
Hmm, yeah it does seem like the response will be either all of mempool (rather than an increment) or no response at all if the request's mempool ID is current.